src/Compilers/CSharp/Test/Semantic/Semantics/NativeIntTests.cs

```diff
@@ -549,39 +630,94 @@ public void SizeOf_04()
                 Diagnostic(ErrorCode.ERR_NotConstantExpression, "sizeof(nuint)").WithArguments("Program.D").WithLocation(6, 19));
         }
 
-        // PROTOTYPE: PEVerify: "[ : MyInt::ToString][mdToken=0x6000005][offset 0x00000001] Cannot change initonly field outside its .ctor."
-        // CodeGenerator.EmitCallExpression() is not expecting a System.ValueType without an overload of ToString().
-        [Fact(Skip = "PEVerify failure")]
+        [Fact]
         public void ReadOnlyField_VirtualMethods()
```

Comment:
>ReadOnlyField_VirtualMethods [](start = 20, length = 28)

I think it would be good to also test delegate creations with these methods. Also, expression trees with calls and delegate creations.